docs/providers/aws/guide/deploying.md

```diff
@@ -44,6 +44,16 @@ The Serverless Framework translates all syntax in `serverless.yml` to a single A
 - The CloudFormation Stack is updated with the new CloudFormation template.
 - Each deployment publishes a new version for each function in your service.
 
+### Deployment method
+
+Since v3 release, by default, Framework uses [change sets](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-changesets.html) to update CloudFormation stacks. Alternatively, it is also possible to update stacks [directly](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-direct.html), which might be faster than changesets-based method in a lot of cases. In order to use `direct` method, you need to configure it explicitly:
```

Comment:
Some suggestions to answer the following questions users may have:

- why should I read this section? ("deployments are faster" is the important part, I'd put it in bold and more visible)
- what's the downside of switching? (none if you don't use changesets)
- I'm not sure, should I switch? (the answer is "yes" for most people)
- if I switch, am I diverging from Framework defaults and creating work/risks for myself? (no, because it will be the default in the future)

```suggestion
Since Serverless Framework v3, deployments are done using [CloudFormation change sets](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-changesets.html). It is possible to use [CloudFormation direct deployments](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-direct.html) instead.

Direct deployments **are faster** and have no downsides (unless you specifically use the generated change sets). They will become the default in Serverless Framework 4.

You are encouraged to enable direct deployments via the `deploymentMethod` option:
```

Reply:
thanks @mnapoli